test/utils/TestUtils.hs

```diff
@@ -202,4 +201,88 @@ stackFileContents resolver = unlines
   , "flags: {}"
   , "extra-package-dbs: []"
   ]
+
```

Comment:
If the block below originated elsewhere, provide details/credit.

Reply:
I had added an attribution to the fork at `xmlFormatter`, is this enough?
https://github.com/haskell/haskell-ide-engine/blob/fb686fa6d5f288ede6235bc31090b81302484fd1/test/utils/TestUtils.hs#L223-L227